docs/sources/write/front-matter/index.md

```diff
@@ -49,6 +49,86 @@ The following headings describe what each element does and provides guidelines f
 
 Use to create redirects from the previous URL to the new URL when a page changes or moves. As a best practice, when you rename or move files, you should create an alias with a reference to the previous URL path to create a redirect from the old URL to the new URL. For more information, refer to [Hugo aliases](#hugo-aliases). In some cases, for example when you have deleted content or split a file into multiple topics, it may not be possible to create an alias for the moved content.
```

Comment:
Good bot.

```suggestion
Use to create redirects from the previous URL to the new URL when a page changes or moves.
As a best practice, when you rename or move files, you should create an alias with a reference to the previous URL path to create a redirect from the old URL to the new URL.
In some cases, for example when you have deleted content or split a file into multiple topics, it may not be possible to create an alias for the moved content.
```